File: JavaScript/Ex.3.js
let vetor = [10]

for(let n = 0; n < 10; n++){
  vetor[n] = prompt("Insira um número: ")
}
for (let n = 0; n < 10; n++) {
  if (vetor[n] % 4 == 0){
    console.log("[" + vetor[n] + "]")
  }
}

File: C#/Ex.3.cs
using System;

class Program {
  public static void Main (string[] args) {
    int[] numero = new int[10];
    int n;

    for(n = 0; n < 10; n++){
      Console.WriteLine("Insira um número: ");
      numero[n] = Convert.ToInt32(Console.ReadLine());
    }
    for(n = 0; n < 10; n++){
      if(numero[n] % 4 == 0){
      Console.WriteLine($"[{numero[n]}]");
      }
    }
  }
}